views/postgres/show.erb

```diff
@@ -194,7 +194,75 @@
                   5432
                 </td>
                 <td class="relative whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-6">
-                  <%== render("components/form/submit_button", locals: { text: "Create" }) %>
+                  <%== render("components/form/submit_button", locals: { text: "Create", extra_class: "firewall-rule-create-button" }) %>
+                </td>
+              </form>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
+  <% end %>
+  <!-- Metric Destination Card -->
+  <% if Authorization.has_permission?(@current_user.id, "Postgres:edit", @pg[:id]) %>
```

Comment:
Could you please add a screenshot to show what it looks like?

Reply:
Added